### PR TITLE
[WIP] Fix 4483 Make sure place exposure works with population

### PR DESCRIPTION
**Pull request by ismailsunni**
### What does it fix?
* Ticket: #4483 
* Funded by: DFAT
* Description: 
   - Enable keyword wizard to set population field for place exposure
   - Fix the flow for field mapping / demographic mapping in the keyword wizard
   - Add unit test

We still need to fix the reporting, since it doesn't look good (mixing between place and the number of population, not showing the breakdown like in the population one). See image below:

screen shot 2017-11-07 at 09 22 02
screen shot 2017-11-07 at 09 22 15

### Checklist:
- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [x] Unit test for new code added
- [x] Request someone to review or test your PR

### CODE REVIEW

File: safe/gui/tools/wizard/test/test_keyword_wizard.py [hunk 1]
```diff
@@ -1786,6 +1788,149 @@ def test_exposure_multi_fields_existing_keyword(self):
...
         self.assertDictEqual(byteify(real_keywords), byteify(expected_keyword))
 
+    def test_exposure_place_population(self):
+        """Test for place with population exposure."""
+        layer = load_test_vector_layer(
+            'gisv4',
+            'exposure',
+            'places.geojson',
+            clone_to_memory=True)
+        self.assertIsNotNone(layer)
```
Comment (Gustry) [inline safe/gui/tools/wizard/test/test_keyword_wizard.py]: No worries, load_test_vector_layer will raise an exception if the layer is not valid ;-)

File: safe/test/data/gisv4/exposure/places.geojson [hunk 2]
```diff
@@ -2,17 +2,13 @@
...
-{ "type": "Feature", "properties": { "exposure_id": 9, "exposure_type": "secret_jail", "fake_field": 666, "absolute": 15, "ratio": 15 }, "geometry": { "type": "Point", "coordinates": [ 106.756827233607638, -6.119749021099483 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 10, "exposure_type": "city", "fake_field": 666, "absolute": 5, "ratio": 2 }, "geometry": { "type": "Point", "coordinates": [ 106.815892813868103, -6.262956479734641 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 11, "exposure_type": null, "fake_field": 666, "absolute": 0, "ratio": 0 }, "geometry": { "type": "Point", "coordinates": [ 106.801427773804321, -6.173081854759827 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 11, "exposure_type": "unknown", "fake_field": 666, "absolute": 0, "ratio": 2 }, "geometry": { "type": "Point", "coordinates": [ 106.893801479188284, -6.118605054148777 ] } }
+{ "type": "Feature", "properties": { "Name": "Alubarna", "Type": "Capital City", "Population": 1000000, "Female": 270000, "Male": 730000 }, "geometry": { "type": "Point", "coordinates": [ 106.752992014058449, -6.171999008052889 ] } },
```
Comment (Gustry) [inline safe/test/data/gisv4/exposure/places.geojson]: You didn't like hogwarts? ;-)

Comment (lucernae) [reply safe/test/data/gisv4/exposure/places.geojson]: He is inserting One Piece reference :v

Comment (ismailsunni) [reply safe/test/data/gisv4/exposure/places.geojson]: :D

**Comment by ismailsunni** [comment]
> Wahou, the IF is working by itself, just because you updated the wizard. Nice.
LGTM

Nice, isn't it? :D 

But, we still need to fix the reporting.

I am merging this...

File: safe/test/data/gisv4/exposure/places.geojson (hunk 2 above)
Comment (ismailsunni) [reply safe/test/data/gisv4/exposure/places.geojson]: I was thinking to use Asgard or Sakaar, but perhaps those are too obvious.

**Pull request merged**